docs/maintain/maintain-guides-how-to-validate-polkadot.md

```diff
@@ -403,7 +403,14 @@ validator mode right away:
 
 :::tip Use Warp syncing for the fast syncing of a pruned node
 
-`./target/production/polkadot --sync warp` imports the latest blockchain state and proofs.
+`./target/production/polkadot --sync warp`
+
+Warp sync initially downloads the whole state of the latest finalized block, and the finality proofs 
```

Comment:
It first downloads the finality proofs and validates them and then downloads the state of the latest finalized block.

Reply:
Edited it. Probably need to suggest this change to the accepted answer on Substrate StackExchange as well https://substrate.stackexchange.com/questions/334/what-kinds-of-sync-mechanisms-does-substrate-implement